modules/migrations/gitlab.go

```diff
@@ -215,6 +215,19 @@ func (g *GitlabDownloader) GetMilestones() ([]*base.Milestone, error) {
 	return milestones, nil
 }
 
+func (g *GitlabDownloader) normalizeColor(val string) string {
+	val = strings.TrimLeft(val, "#")
+	val = strings.ToLower(val)
+	if len(val) == 3 {
+		c := []rune(val)
+		val = fmt.Sprintf("%c%c%c%c%c%c", c[0], c[0], c[1], c[1], c[2], c[2])
```

Comment:
if somebody know a better func to convert `abc` -> `aabbcc` pleace tell me :)

Reply:
```go
val := "abc"
doubleVal := make([]rune,0, len(val)*2)
for _, v := range val {
	doubleVal = append(doubleVal, v, v)
}
val = string(doubleVal)
```

Reply:
but it's not really better